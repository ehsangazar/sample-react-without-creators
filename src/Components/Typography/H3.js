import React from 'react'
import styled from 'styled-components'

const H3Styled = styled.h3`
  font-size: 14px;
  margin:0;
`

const H3 = ({children, ...props}) => <H3Styled {...props}>{children}</H3Styled>

export default H3