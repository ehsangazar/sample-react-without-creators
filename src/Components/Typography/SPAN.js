import React from 'react'
import styled from 'styled-components'

const SPANStyled = styled.span`
  font-size: 12px;
  color: #ababab;
`

const SPAN = ({children, ...props}) => <SPANStyled {...props}>{children}</SPANStyled>

export default SPAN