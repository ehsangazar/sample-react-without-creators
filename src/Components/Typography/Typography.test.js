import React from 'react';
import SPAN from './SPAN';
import H3 from './H3';
import renderer from 'react-test-renderer';

test('H3 should render correctly', () => {
  const component = renderer.create(
    <H3>Test</H3>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
test('SPAN should render correctly', () => {
  const component = renderer.create(
    <SPAN>Test</SPAN>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});