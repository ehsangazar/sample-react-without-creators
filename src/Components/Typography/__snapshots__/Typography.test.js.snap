// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 should render correctly 1`] = `
<h3
  className="sc-bwzfXH fawXov"
>
  Test
</h3>
`;

exports[`SPAN should render correctly 1`] = `
<span
  className="sc-bdVaJa gnuIcu"
>
  Test
</span>
`;
